.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://github.com/googleforgames/agones/blob/master/CONTRIBUTING.md and developer guide https://github.com/googleforgames/agones/blob/master/build/README.md
+2. Please label this pull request according to what type of issue you are addressing.
+3. Ensure you have added or ran the appropriate tests for your PR: https://github.com/googleforgames/agones/blob/master/build/README.md#testing-and-building
+-->
+
+**What type of PR is this?**
+> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
+>
+> /kind breaking
+> /kind bug
+> /kind cleanup
+> /kind design
+> /kind documentation
+> /kind feature
+> /kind hotfix
+> /kind release
```

Comment:
Since this is only used by a maintainer cutting a release, I don't think it should be in the list for folks to opt into. It's likely to cause more mistakes than it helps us.

Reply:
Yes, it makes sense. I can remove "What type of PR is this?" section and leave others as is, ok?

Reply:
I was commenting on line 18, not the whole section. I think asking contributors to self-label PRs is actually nice.

Reply:
Ok